docs/book/stacks-and-components/component-guide/model-deployers/model-deployers.md

```diff
@@ -87,46 +82,52 @@ zenml model-deployer register seldon --flavor=seldon \
-   that is already serving a previous version of the same model instead of creating a new model server for every new
-   model version. Every model server that the Model Deployer provisions externally to deploy a model is represented
-   internally as a `Service` object that may be accessed for visibility and control over a single model deployment. This
-   functionality can be consumed directly from ZenML pipeline steps, but it can also be used outside the pipeline to
-   deploy ad-hoc models. See the [seldon_model_deployer_step](https://sdkdocs.zenml.io/latest/integration_code_docs/integrations-seldon/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step) for an example of using the Seldon Core Model Deployer to deploy a model inside a ZenML pipeline step.
-3. Acts as a registry for all Services that represent remote model servers. External model deployment servers can be
-   listed and filtered using a variety of criteria, such as the name of the model or the names of the pipeline and step
-   that was used to deploy the model. The Service objects returned by the Model Deployer can be used to interact with
-   the remote model server, e.g. to get the operational status of a model server, the prediction URI that it exposes, or
-   to stop or delete a model server:
+* Lifecycle Management: Provides mechanisms for comprehensive lifecycle management of model servers, including the ability to start, stop, and delete model servers, as well as to update existing servers with new model versions, thereby optimizing resource utilization and facilitating continuous delivery of model updates. Some core methods that can be used to interact with the remote model server include:
+
+`deploy_model` - Deploys a model to the serving environment and returns a Service object that represents the deployed model server.
+`find_model_server` - Finds and returns a list of Service objects that represent model servers that have been deployed to the serving environment, the 
+services are stored in the DB and can be used as a reference to know what and where the model is deployed.
+`stop_model_server` - Stops a model server that is currently running in the serving environment.
+`start_model_server` - Starts a model server that has been stopped in the serving environment.
+`delete_model_server` - Deletes a model server from the serving environment and from the DB.
+
+{% hint style="info" %}
+ZenML uses the Service object to represent a model server that has been deployed to a serving environment. The Service object is saved in the DB and can be used as a reference to know what and where the model is deployed. The Service object consists of 2 main attributes, the `config` and the `status`. The `config` attribute holds all the deployment configuration attributes required to create a new deployment, while the `status` attribute holds the operational status of the deployment, such as the last error message, the prediction URL, and the deployment status.
```

Comment:
Wondering whether to give the details of where the service object is being saved... we don't talk about that elsewhere in the documentation. Wonder whether it's confusing for a user. Maybe better to say something like 'saved on the ZenML Server' or something, or 'saved as metadata on the ZenML server' etc?